private4/index.php

```diff
@@ -245,8 +239,7 @@ function lastOne() {
 
 
 <!-- OSD -->
-<script src="/openseadragon/openseadragon.min.js"></script>
-<script src="/openseadragon/zoomifytilesource.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/openseadragon@2.3/build/openseadragon/openseadragon.min.js"></script>
```

Comment:
This is high speed delivery of the latest OpenSeadragon, which includes Zoomify.